### ISSUE TITLE
Cannot run `check` in CLI

### ISSUE THREAD

**Issue by MartinZikmund** [all-contributors/all-contributors#589]
**Describe the bug**

I tried following the documentation, but after running `init` and then immediately `check` only get:

```
❯ yarn all-contributors check
yarn run v1.22.17
$ C:\Dev\uno\node_modules\.bin\all-contributors check
Only absolute URLs are supported
error Command failed with exit code 1.
```

**To Reproduce**
1. Install CLI
2. Run `init`
3. Run `check`

**Expected behavior**

Should not end with error.

**Comment by thedavidprice** [comment]
I'm seeing this behavior on one of our repos but not all. For the repo https://github.com/redwoodjs/redwood this fails:
```
"projectName": "redwood"
```

But for the repo https://github.com/redwoodjs/learn.redwoodjs.com, this works:
```
  "projectName": "learn.redwoodjs.com",
```

I've been using this for over a year and haven't made changes or upgrades to All Contributors. Assuming something changed regarding permissions/settings for the repo. Have no idea what that could be (yet)?

**Comment by thedavidprice** [comment]
node-fetch issue. Related code is over here: https://github.com/all-contributors/cli

Did a quick check for now and didn't find anything obvious. Likely something broke with more recent versions of node-fetch. Will need more time to resolve this.

**Comment by Berkmann18** [comment]
@thedavidprice Do you know if there was a bug fix for that?

**Comment by tenshiAMD** [comment]
@MartinZikmund this seems irrelevant now. Please check the latest version. Closing this now so feel free to post an issue if the error still occurs.

**Comment by thedavidprice** [comment]
@Berkmann18 we patched directly

I do recommend checking if the latest version resolves this.

cc @jtoar Looks like version was released a few hours ago. We'll need to coordinate upgrading along with un-patching if so. TBD

**Comment by Berkmann18** [comment]
Yeah, it's worth checking with the updated versions to see if that happens.
